src/Interpreters/Context.h

```diff
@@ -15,6 +15,8 @@
 #include <Common/isLocalAddress.h>
 #include <base/types.h>
 #include <Storages/MergeTree/ParallelReplicasReadingCoordinator.h>
+#include <rocksdb/db.h>
+#include <rocksdb/table.h>
```

Comment:
I think that it should be still possible to build ClickHouse without RocksDB and this code should be under the build flag. Also I noticed that cache is added unconditionally without any setting. There should exist setting that disables cache and it should be disabled for the first time until we will not be sure that it doesn't have any bugs. It's important, because in case of some problems the server wouldn't start at all and it will be very serious issue.

If cache is enabled by setting and ClickHouse was built without RocksDB, it will be ok to write the warning and fallback to reading files from `IDisk` or even throw exception and refuse to start.

Reply:
Thanks for your valuable advice. I would try.

Reply:
Already done.